charts/rekor/templates/server/configmap-sharding.yaml

```diff
@@ -0,0 +1,10 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: {{ .Values.server.sharding.name }}
```

Comment:
for consistency, can you change to generate the name similar to existing configmap, with template "rekor.config.

https://github.com/sigstore/helm-charts/blob/main/charts/rekor/templates/server/configmap.yaml#L4
https://github.com/sigstore/helm-charts/blob/main/charts/rekor/templates/_helpers.tpl#L410

Reply:
👍🏽